packages/babel-preset/index.js

```diff
@@ -19,6 +21,14 @@ module.exports = api => ({
   ],
   plugins: [
     api.env('test') ? false : ['babel-plugin-add-import-extension'],
+    api.env('test')
+      ? [
+          // This is needed to replace import.meta w/ process in Jest
+          // Jest does not play nicely w/ ESM and Vite uses import.meta
+          // import.meta is only avaialable in ESM
+          path.resolve(__dirname, 'importMetaEnvPlugin'),
+        ]
+      : false,
```

Comment:
You could just write this in one ternary instead of two, e.g.
```suggestion
    api.env('test')
      ? [
          // This is needed to replace import.meta w/ process in Jest
          // Jest does not play nicely w/ ESM and Vite uses import.meta
          // import.meta is only avaialable in ESM
          path.resolve(__dirname, 'importMetaEnvPlugin'),
        ]
      : ['babel-plugin-add-import-extension'],
```

Might be worth explaining why we don't need babel-plugin-add-import-extension for Jest

Reply:
I don't know why we ignore the add import extnsion plugin in test, but it causes tests to error so I'll add that comment